app/assets/stylesheets/contact_forms/new.scss

```diff
@@ -9,33 +9,40 @@
 .field-wrapper {
   margin-top: 15px;
   position: relative;
-}
 
-.field-wrapper label {
-  border-radius: 3px;
-  color: #AAA;
-  font-style: italic;
-  font-size: 15px;
-  left: 4px;
-  padding: 4px 6px;
-  position: absolute;
-  top: 4px;
-  transition: -webkit-transform .15s, opacity .2s, top .15s, left .15s, background-color .15s, color .15s;
-}
+  label {
+    @include transition(opacity 0.2s, top 0.15s, left 0.15s, background-color 0.15s, color 0.15s);
+    @include transition-property(transform);
+    @include transition-duration(0.15s);
+    border-radius: 3px;
+    color: #AAA;
+    font-style: italic;
```

Comment:
Properties should be ordered border-radius, color, font-size, font-style, left, padding, position, top